docs/ops/activation/HSwish_4.md

```diff
@@ -0,0 +1,50 @@
+## HSwish <a name="HSwish"></a> {#openvino_docs_ops_activation_HSwish_4}
```

Comment:
Please, update the docs/ops/opset4.md file.

Reply:
Thanks, done